### PR TITLE
added endpoint for file upload

### PR DESCRIPTION
**Pull request by NicoleSauer**
I was using Postman to verify the file upload was successful.

- **test_cv.pdf** is included for testing the endpoint with Postman
- I also included my exported Postman workspace, so everyone should be able to import it and test the endpoint 

In Postman. there could be a problem with the location of the **test_cv.pdf** which is attached to the body of the POST request. Maybe it needs to be adjusted manually.

Bildschirmfoto 2023-11-27 um 17 13 50

Maybe we should also talk about what software to use for API testing (Postman, Insomnia, ...)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -163,4 +163,8 @@ cython_debug/
 *.ppm
 
 # vector store
```
Comment (ScottPalme) [inline .gitignore:165]: Could you do me a favour and add `.idea` because of pycharm

File: src/apis/applicant_api.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        if 'pdfFile' not in request.files:
+            return 'No file part found in POST request.', 400
+
+        pdf_file = request.files['pdfFile']
+
+        # Handle the uploaded file (save, process, etc.)
+        print('File name: ' + pdf_file.filename)
+
+        # Change filename to the applicant id
+        # TODO: Generate UUID for applicant id somewhere
```
Comment (ScottPalme) [inline src/apis/applicant_api.py:26]: The uuid of the applicant is set in the BusinessObject abstract class when initialising a Applicant instance. If this resolves the TODO and the concern.

File: .gitignore (hunk 1 above)
Comment (NicoleSauer) [reply .gitignore:165]: Sure, no problem. I added it in the latest commit.

File: src/apis/applicant_api.py (hunk 2 above)
Comment (NicoleSauer) [reply src/apis/applicant_api.py]: Yes, this resolves my TODOs. Since these classes are not on the main branch yet, I forgot about them.

**Pull request merged**